testing/tests/DevExpress.ui/notify.tests.js

```diff
@@ -171,8 +173,18 @@ QUnit.module('notify', {
 
             const stackStyles = $(`.${TOAST_STACK}`)[0].style;
 
-            assert.equal(stackStyles.bottom, `${position.bottom}px`);
-            assert.equal(stackStyles.right, `${position.right}px`);
+            assert.strictEqual(stackStyles.bottom, `${position.bottom}px`);
+            assert.strictEqual(stackStyles.right, `${position.right}px`);
+            this.clock.tick(100);
```

Comment:
why do we need this tick?
If we really need it it's a good idea to move it to `afterEach`

Reply:
because of displayTime: 100
moved to aferEach